Sources/Core/Shared/OperationQueue.swift

```diff
@@ -197,7 +197,7 @@ public extension OperationQueue {
 
      - returns: The main queue
      */
-    public override class func main() -> OperationQueue {
+    public class func main() -> OperationQueue {
```

Comment:
I see - so, need to check `Foundation.OperationQueue` to see if this has changed to a static property or just been removed...